aexpect/client.py

```diff
@@ -1093,7 +1093,7 @@ def cmd_output_safe(self, cmd, timeout=60):
                 o += self.read_up_to_prompt(0.5)
                 success = True
                 break
-            except ExpectError, e:
+            except ExpectError as e:
```

Comment:
I think we should rather ask @will-Do and @xutian. Is it OK to use python2.6 features already or do you still require to stick with the older one? Particularly things like this, are they OK?